### PR TITLE
[RW-1088][RW-1089][RW-1090][risk=no] Skin Attribute, Modifiers, Wizard according to style guide 

### PR DESCRIPTION
**Pull request by smazumd**
Skin Attribute, Modifiers, Wizard according to style guide 

1. Below Screen Shot shows the Wizard according to style guide

screen shot 2018-08-14 at 9 09 57 am

2. Below Screenshot is about Modifiers according to style guide
screen shot 2018-08-14 at 9 10 16 am

3. Below screenshot is with Clarity Calendar for Modifier

screen shot 2018-08-14 at 9 10 35 am

4.Below Screenshot is about Attribute according to style guide

screen shot 2018-08-14 at 9 11 47 am

### CODE REVIEW

File: ui/src/app/cohort-search/modifier-page/modifier-page.component.html [hunk 1]
```diff
@@ -1,36 +1,80 @@
...
+      <span *ngIf="preview.get('requesting')" class="spinner spinner-inline"> Loading...   </span>
+    </div>
+    <div class="col-lg-8 text-padding" *ngIf="preview.get('count') >= 0">
+      <div class="result-text">
+        Results
+      </div>
+      <div>
+        Number Participants:
+        <span class="text-bold" *ngIf="!preview.get('requesting')">
+             {{ preview.get('count', 0) }}
```
Comment (dolbeew) [inline ui/src/app/cohort-search/modifier-page/modifier-page.component.html]: Add number pipe here also

File: ui/src/app/cohort-search/attributes-page/attributes-page.component.html [hunk 2]
```diff
@@ -13,47 +13,68 @@
...
+          <span *ngIf="preview.get('requesting')" class="spinner spinner-inline"> Loading...   </span>
+        </div>
+        <div class="col-lg-8 text-padding" *ngIf="preview.get('count') >= 0">
+          <div class="result-text">
+            Results
+          </div>
+           <div >
+              Number Participants:
+             <span class="text-bold" *ngIf="!preview.get('requesting')">
+             {{ preview.get('count', 0) }}
```
[the remark this answers is not in the document]
Comment (smazumd) [reply ui/src/app/cohort-search/attributes-page/attributes-page.component.html]: Working on it.

File: ui/src/app/cohort-search/modifier-page/modifier-page.component.html (hunk 1 above)
Comment (smazumd) [reply ui/src/app/cohort-search/modifier-page/modifier-page.component.html]: Working on it.

File: ui/src/app/cohort-search/attributes-page/attributes-page.component.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (smazumd) [reply ui/src/app/cohort-search/attributes-page/attributes-page.component.html]: done

File: ui/src/app/cohort-search/modifier-page/modifier-page.component.html (hunk 1 above)
Comment (smazumd) [reply ui/src/app/cohort-search/modifier-page/modifier-page.component.html]: Done

**Comment by dolbeew** [comment]
The sidebar for selections in the wizard gets cutoff on smaller screens. This is on my laptop:
screen shot 2018-08-14 at 1 03 57 pm

**Comment by lstrano-broad** [comment]
Updated screens look good - thx Shanta!

**Pull request merged**